tests/robot-tests/tests/general_public/notifications_absence.robot

```diff
@@ -35,5 +35,6 @@ Sign up for email alerts
     user clicks button    Subscribe
 
     # EES-1265
-    user waits until h1 is visible    Subscribed    %{WAIT_LONG}
+    #user waits until h1 is visible    Subscribed    %{WAIT_LONG}
+    user waits until page contains    Subscribed    %{WAIT_SMALL}
```

Comment:
I think we need to keep the `WAIT_LONG` here. The ticket EES-1265 suggests that if nobody has used notify for a while, the response can be very slow, so to ensure this tests passes if the response from notify does take a long time, we need a longer wait.